alerts.md

```diff
@@ -233,6 +233,34 @@ Example Helm values.yaml:
           threshold: 5              # Send Alert if health scores changes by 5 or more
 ```
 
+## Configuring Alerts in the Kubecost UI
+
+### Cluster Health and Diagnostic Alerts
+Cluster Health Alerts and Diagnostic Alerts work differently from other alert types. While other alerts monitor cost data for cost or efficiency anomalies, Health and Diagnostics montior the health of Kubecost itself, as well as the health of the cluster running Kubecost. The UI treats these alert types as "on" or "off", managing a single instance of each, and allowing the settings of this single instance to be adjusted.
+
+![Kubecost Health Alerts](images/health_and_diagnostics_alert_controls.png)
+
+NOTE: As of this writing, there is no validation around savings Cluster Health Alerts. If a Health Alert configuration is invalid, it will appear to save, but will not actually take effect. Please check carefully that the alert has a Window and Threshold properly specified.
```

Comment:
"savings Cluster Health Alerts" sounds confusing is that a typo?

Is there not way to verify if a health alert is valid? Why not suggest sending a test alert?

Reply:
Totally a typo. I'll check into how we might validate.